core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -30,39 +31,83 @@
 import io.micrometer.core.instrument.MeterRegistry;
 import io.micrometer.core.instrument.Tag;
 import io.micrometer.core.instrument.Timer;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
 
 /**
  * Provides utilities for accessing {@link MeterRegistry}.
  */
 public final class MoreMeters {
 
+    private static DistributionStatisticConfig distributionStatisticConfig;
+
+    public static void withConfig(DistributionStatisticConfig config) {
+        distributionStatisticConfig = config;
+    }
+
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
     /**
-     * Returns a newly-registered {@link DistributionSummary} with percentile publication configured.
-     */
-    public static DistributionSummary summaryWithDefaultQuantiles(MeterRegistry registry,
-                                                                  String name, Iterable<Tag> tags) {
+     * Returns a newly-registered {@link DistributionSummary}.
```

Comment:
Append: `with the current {@linkplain #distributionStatisticConfig() distribution statistic configuration}.`